pipeline/Dockerfile

```diff
@@ -9,14 +9,15 @@ RUN sudo apt-get update && \
     libgmp-dev \
     libpq-dev \
     graphviz \
-    capnproto \
     libsqlite3-dev \
+    capnproto \
     libcapnp-dev
 
 WORKDIR /mnt/project
 
 # Build dependencies.
 COPY --chown=opam:opam pipeline.opam pipeline.opam
+RUN sudo mv /usr/bin/opam-2.1 /usr/bin/opam
```

Comment:
Also using 2.1 opam, doesn't really make a difference since we are not using any 2.1 features, yet